src/main/resources/pdiresources/concept/preparation.ktr

```diff
@@ -1366,7 +1366,7 @@
     <single_statement>N</single_statement>
     <replace_variables>N</replace_variables>
     <quoteString>N</quoteString>
-    <sql>CREATE TABLE `old_concept` (
+    <sql>CREATE TABLE IF NOT EXISTS `old_concept` (
```

Comment:
@cfaife  What happens if the previous run failed and you need to run again. Will this table already have data in it, and could it be a problem?

Reply:
Morning please consider other PR